modules/arm_plugin/docs/arm_opset.md

```diff
@@ -184,7 +184,7 @@ The semantics match corresponding nGraph operation classes declared in `namespac
 * `'ConvertLike'` layer is supported configuration like  `'Convert'`.
 * `'DepthToSpace'` layer is supported 4D tensors only and for `BLOCKS_FIRST` of `'mode'` attribute.
 * `'Equal'` doesn't support `broadcast` for inputs.
-* `'Gather'` layer is supported constant scalar or 1D indices axes only.
+* `'Gather'` layer is supported constant scalar or 1D indices axes only. Layer is supported as  via arm_compute library with non negative indices else use reference relaization. 
```

Comment:
IMO
"Layer is supported via arm_compute library for non negative indices and via reference implementation otherwise."
looks better

Reply:
@terfendail done